### PR TITLE
[QPE-318] Applied build pipeline magic

### PR DESCRIPTION
**Pull request by marols**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/qlik-radar-chart
+  docker:
+    - image: circleci/node:stretch
+  environment:
+    GITHUB_ORG: "qlik-oss"
+    GITHUB_REPO: "br.com.clever.wordcloud"
```
Comment (kristofferlind) [inline .circleci/config.yml]: wrong repo

File: README.md [hunk 2]
```diff
@@ -16,37 +16,55 @@ Please try the new version, and leave feedback, as we endeavour to improve our e
...
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+# License
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+Copyright © 2015 Brian Booden & Matthieu Burel
```
Comment (kristofferlind) [inline README.md]: License file has both original authors and qlik as copyright text, possibly just say MIT with a link to the license file?

File: .circleci/config.yml (hunk 1 above)
Comment (marols) [reply .circleci/config.yml]: Fixed

File: README.md (hunk 2 above)
Comment (marols) [reply README.md]: Fixed

File: package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "qlik-radar-chart",
+  "version": "0.0.1",
+  "description": "Qlik Sense Extension Radar chart",
+  "keywords": "word cloud qliksense extension",
```
Comment (kristofferlind) [inline package.json]:
```suggestion
  "keywords": "radar chart qliksense extension",
```

**Pull request merged**